tests/test_pipeline_images.py

```diff
@@ -31,18 +31,13 @@
         skip_pillow = None
 
 
-def _mocked_download_func(request, info):
-    response = request.meta.get('response')
-    return response() if callable(response) else response
-
-
 class ImagesPipelineTestCase(unittest.TestCase):
 
     skip = skip_pillow
 
     def setUp(self):
         self.tempdir = mkdtemp()
-        self.pipeline = ImagesPipeline(self.tempdir, download_func=_mocked_download_func)
```

Comment:
Why is this no longer needed?

Reply:
It is unused. I tried raising an exception from the download function, and the tests passed nonetheless, indicating that it is not actually used.